understanding/20/images-of-text.html

```diff
@@ -133,7 +133,7 @@ <h2>Examples of Images of Text</h2>
       <h2>Resources for Images of Text</h2>
       <ul>
         <li><a href="https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Text_styling/Fundamentals">Fundamental text and font styling</a></li>
-        <li><a href="https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Text_styling/Web_fonts">MDN Web fonts</a></li>
+        <li><a href="https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Text_styling/Web_fonts">MDN web fonts</a></li>
```

Comment:
This use of "web" is in context of cited title, so could go either way.

Reply:
```suggestion
        <li><a href="https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Text_styling/Web_fonts">MDN: Web fonts</a></li>
```